modules/lead/product-aws/aws.tf

```diff
@@ -1,31 +1,46 @@
 resource "aws_codebuild_project" "codebuild_build" {
   for_each = var.pipelines
 
-  name          = "${var.product_name}-${each.value.repo}-build"
+  name          = "${each.value.repo}-build"
```

Comment:
why did the product name get removed from the name?  is it okay to have two `aws_codebuild_project` resources with the same name?

Reply:
I originally removed it since sparky import was appending the product name to the repo name also so it was duplicating the name, I added it back since that should just be fixed in the SDM